src/components/EditCoursePage/EditCoursePage.test.jsx

```diff
@@ -948,6 +949,7 @@ describe('EditCoursePage', () => {
       subjects: ['basket-weaving'],
       syllabus_raw: null,
       title: 'demo4004',
+      topics: [],
```

Comment:
nit: please add some tests with some values of topics, or you can do this in your subsequent PR